libs/SalesforceSDKCore/SalesforceSDKCore/Classes/OAuth/SFOAuthCoordinator.m

```diff
@@ -887,32 +879,6 @@ - (void) updateCredentials:(NSDictionary *) params {
 
 }
 
-- (void)configureWebUserAgent
-{
-    if (self.userAgentForAuth != nil) {
-        NSString *origWebUserAgent = [[NSUserDefaults msdkUserDefaults] objectForKey:kOAuthUserAgentUserDefaultsKey];
-        if (origWebUserAgent != nil) {
-            self.origWebUserAgent = origWebUserAgent;
-        }
-        
-        NSDictionary *userAgentDict = @{ kOAuthUserAgentUserDefaultsKey: self.userAgentForAuth };
-        [[NSUserDefaults msdkUserDefaults] registerDefaults:userAgentDict];
-    }
-}
-
-- (void)resetWebUserAgent
-{
-    if (self.userAgentForAuth != nil) {
-        // If the current web user agent has not changed from the one we set, reset it.  Otherwise, assume it's
-        // already been altered out of band, and we shouldn't touch it.
-        NSString *currentWebUserAgent = [[NSUserDefaults msdkUserDefaults] objectForKey:kOAuthUserAgentUserDefaultsKey];
```

Comment:
👍 This was the legacy way to do it for `UIWebView` if I'm not mistaken.

Reply:
Thats correct!